frame/dex-router/src/lib.rs

```diff
@@ -94,8 +94,6 @@ pub mod pallet {
 		NoRouteFound,
 		/// Unexpected node found.
 		UnexpectedNodeFound,
```

Comment:
not sure I get what error means? what node is and why as user I care about nodes? i care about pools, is that rout from A to B not found? but there is NoRouteFound